adr/0014-home-assistant-supervised.md

```diff
@@ -20,14 +20,25 @@ This method is considered advanced and should only be used if one is an expert i
 
 ### Supported Operating System, System dependencies and versions
 
-Docker CE (Community Edition) is the only supported containerization method for Home Assistant Supervised. We only support FHS 3.0 on the host file system.
+Docker CE (Community Edition) is the only supported containerization method for Home Assistant Supervised. It can be installed in many different ways, one of them is using the script here https://get.docker.com/ another is adding the repo manually like described here https://www.linode.com/docs/guides/how-to-install-docker-ce-on-debian-10/. We only support FHS 3.0 on the host file system.
```

Comment:
This is not a guide providing installation instructions, but this is an architectural decision record. This does not belong here.

Reply:
I agree when you put it like that, but it's linked to here as "requirements", https://github.com/home-assistant/supervised-installer, so maybe that page needs re-wording?

As an example, atm the insaller checks for `docker`, and will pass even if the user installed the apt upstream non community edition. I just thought it would make it a bit clearer to explain what is checked against with regards to deps.

Reply:
It is linked to the requirements that are listed below.

Your text adds third-party installation instructions, which is not what this document is for.